src/main/java/uk/gov/hmcts/reform/jobscheduler/services/jobs/SchedulerExceptionHandlingHelper.java

```diff
@@ -23,7 +24,4 @@
             throw new JobException(exc.getMessage(), exc);
         }
     }
-
-    private SchedulerExceptionHandlingHelper() {
```

Comment:
Was the presence of this constructor the problem?

Reply:
yeh, because it was empty :|

Reply:
It would probably be satisfied with a comment in the body. This looks like an intentionally hidden constructor.

Reply:
I think we had few discussion around this matter. Lately we are OK to suppress warning rather than keeping private constructor just because

Reply:
Yeh the code does nothing, I can't see the point in keeping it there to satisfy some check style rule or whatever tbh.